TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -13,6 +13,7 @@
 using System.Collections.Concurrent;
 using TypeCobol.Compiler.CodeElements;
 using TypeCobol.Compiler.Nodes;
+using TypeCobol.LanguageServer.TypeCobolCustomLanguageServerProtocol;
```

Comment:
From my point of view (regarding its description), it is a little bit strange that `Workspace `refers to `TypeCobolCustomLanguageServerProtocol `(by the way that introduces a circular reference).
What do you think?

Reply:
You're right, I'll refactor that a bit.